BibBuddy/app/src/main/java/de/bibbuddy/NoteRecyclerViewAdapter.java

```diff
@@ -73,19 +74,17 @@ public NotesViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     return new NotesViewHolder(itemView);
   }
 
-
   /**
-   * Method to setup the custom ViewHolder components for notes.
+   * Method to set up custom ViewHolder components for Notes.
```

Comment:
In line with the suggestions you gave me in #169 I would advise to use active wording here.
```suggestion
   * Set up custom ViewHolder components for Notes.
```